string2pdb/README.md

```diff
@@ -1,21 +1,28 @@
-# DESCRIPTION
-This script extracts filtered interactors for a specified target from the STRING database. 
-It provides the combined score of the confidence of each interaction (**String_score**), 
-along with the three components of the physical network score: **Experimental_score**, **Database_score** and **Text_mining score**. 
-**String_score** is used as the threshold for filtering.
+# Description:
+- This script extracts filtered interactors for a specified target from the STRING database, using the **STRING API**. 
+- It searches only in the **physical** subnetwork of STRING by default, but can be adjusted to parse the entire database (functional network).
+- It provides the combined score of the confidence of each interaction (**String score**), along with 
+the three components of the physical network interaction score: 
+   - **Experimental score**
+   - **Database score**
+   - **Text mining score**. 
+- The **String Score** threshold is adjustable, allowing the user to specify the desired confidence level for extracted interactions. 
+- The **Database Score** can be adjusted as well, to specify the desired level of association in curated databases.
+- The script also maps the STRING identifier of the retrieved interactors to their **primary UniProt ACs**, using the preprocessed 
+STRING human protein alias file (https://stringdb-downloads.org/download/protein.aliases.v12.0/9606.protein.aliases.v12.0.txt.gz).
 
-# REQUIREMENTS
+# Requirements:

```

Comment:
could you also add what we need in terms of data? (i.e. the modified STRING database)

Reply:
can you also upload the script for generating the modified STRING correspondence as well as instructions on what it needs and how to use it? Please remember to add a header with license information and copyright as in this script :)